js/start.md

```diff
@@ -3,12 +3,26 @@
 
 # Getting Started
 
-This page is a guide to quickly build a cloud-powered mobile or web app with AWS Amplify. AWS Amplify includes:
-- a JavaScript library with support for React Native and web frameworks including <br/>React, Angular, and Ionic
-- a style guide including UI components
-- the Amplify CLI with support for managing the serverless backend, web hosting, and codegen
+This page is a guide to quickly build a cloud-powered web or React Native application with AWS Amplify. AWS Amplify includes:
+
+## What Does AWS Amplify Include?
```

Comment:
This section is attempting to delineate the 3 basic components of the Amplify framework (exclusing the console).  I feel like explaining these three pieces up front will help users understand how the framework operates.

Reply:
This is great.